packages/rpc-provider/src/substrate-connect/ScProvider.ts

```diff
@@ -233,6 +249,32 @@ export class ScProvider implements ProviderInterface {
     }
   }
 
+  #resubscribe = (): void => {
+    const promises: any[] = [];
+
+    this.#resubscribeMethods.forEach((subDetails: ActiveSubs): void => {
+      // only re-create subscriptions which are not in author (only area where
+      // transactions are created, i.e. submissions such as 'author_submitAndWatchExtrinsic'
+      // are not included (and will not be re-broadcast)
+      if (subDetails.type.startsWith('author_')) {
+        return;
+      }
+
+      try {
+        const promise: Promise<void> = new Promise((resolve) => {
+          this.subscribe(subDetails.type, subDetails.method, subDetails.params, subDetails.callback).catch((error) => console.log(error));
+          resolve();
+        });
+
+        promises.push(promise);
+      } catch (error) {
+        l.error(error);
+      }
+    });
+
+    Promise.all(promises).catch((err) => console.log(err));
```

Comment:
```suggestion
    Promise.all(promises).catch((err) => l.log(err));
```

Probably want to use the logger here, since it is included.